frontend/src/components/Field/Attachment/AttachmentDropzone.tsx

```diff
@@ -8,19 +8,21 @@ interface AttachmentDropzoneProps {
   inputProps: DropzoneInputProps
   isDragActive: DropzoneState['isDragActive']
   readableMaxSize?: string
+  question?: string
 }
 
 export const AttachmentDropzone = ({
   inputProps,
   isDragActive,
   readableMaxSize,
+  question,
 }: AttachmentDropzoneProps): JSX.Element => {
   const styles = useStyles()
 
   return (
     <>
       <VisuallyHidden>
-        Click to upload file, maximum file size of {readableMaxSize}
+        {question} Click to upload file, maximum file size of {readableMaxSize}
```

Comment:
Is there a reason why we put the question here over including it as an aria-label in the chakra.input below?

Reply:
Oh, because:
1. The question should come before what the user should do.
2. I kind of assumed this was a better place to put it as VisuallyHidden is intended for screen readers and previous engineers chose to. 

Anyways, I tried it out with the following snippet:
```ts
<chakra.input
  aria-label={`${question} Click to upload file, maximum file size of ${readableMaxSize}`}
  {...inputProps}
  data-testid={inputProps.name}
/>
```

On windows, this will just read "button required" as seen in the video below:

https://user-images.githubusercontent.com/37061143/219065430-3e9b504d-0077-4c6c-bea4-aed94ed96e97.mp4

So this probably means NVDA does not pick up the `aria-label` of `chakra.input`. Thus, I'm guessing `VisuallyHidden` was used (by @justynoh I think) for screen readers to pick up and give more informative descriptions to our visually impaired users.

Relatedly, I just read [this article](https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/) that points out the flaws of `aria-label`s and explains when to use `VisuallyHidden` elements.

Reply:
Oh nice! Thanks I realised that aria-labels also don't work as expected all the time hahaha, maybe it has to do with assigning a role to the input or perhaps another option would be to envelop the input+icon field with a label container like in radio & Yes/No.

But seems like visually hidden works well here!